kolibri/core/assets/src/vue/icon-button.vue

```diff
@@ -72,6 +72,10 @@
     padding-right: 8px
     height: 36px
 
+  @media screen and (max-width: $portrait-breakpoint)
+    .icon-button
+      margin: 5px
+
```

Comment:
it would be preferable to do this without setting a media query - is it possible to just always give it a 5px margin?